docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -19,7 +19,11 @@ yarn add aws-amplify @aws-amplify/ui-react
 </TabItem>
 </Tabs>
 
-Wrap your application in `<AmplifyProvider/>` and include default styling:
+After adding the `aws-amplify` and `@aws-amplify/ui-react` dependencies you are now ready to add any of our [components](/components) to your application.
+
+### Theming
```

Comment:
Maybe this should call out that it's only needed for a custom theme in the title itself?
```suggestion
### Custom Theme
```

Reply:
Done